modules/apps/layout/layout-seo-web/src/main/java/com/liferay/layout/seo/web/internal/configuration/FFLayoutTranslatedLanguagesConfiguration.java

```diff
@@ -0,0 +1,33 @@
+/**
+ * Copyright (c) 2000-present Liferay, Inc. All rights reserved.
+ *
+ * This library is free software; you can redistribute it and/or modify it under
+ * the terms of the GNU Lesser General Public License as published by the Free
+ * Software Foundation; either version 2.1 of the License, or (at your option)
+ * any later version.
+ *
+ * This library is distributed in the hope that it will be useful, but WITHOUT
+ * ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS
+ * FOR A PARTICULAR PURPOSE. See the GNU Lesser General Public License for more
+ * details.
+ */
+
+package com.liferay.layout.seo.web.internal.configuration;
+
+import aQute.bnd.annotation.metatype.Meta;
+
+import com.liferay.portal.configuration.metatype.annotations.ExtendedObjectClassDefinition;
+
+/**
+ * @author Alicia García
+ */
+@ExtendedObjectClassDefinition(generateUI = false)
+@Meta.OCD(
+	id = "com.liferay.layout.seo.web.internal.configuration.FFLayoutTranslatedLanguagesConfiguration"
+)
+public interface FFLayoutTranslatedLanguagesConfiguration {
+
+	@Meta.AD(deflt = "false", required = false)
+	public boolean enableFFLayoutTranslatedLanguages();
```

Comment:
Don't forget to add a message for Brian, explaining why we cannot simply use "enabled" here.